third_party/terraform/data_sources/data_source_google_kms_crypto_key_version.go

```diff
@@ -105,7 +105,16 @@ func dataSourceGoogleKmsCryptoKeyVersionRead(d *schema.ResourceData, meta interf
 			return err
 		}
 		log.Printf("[DEBUG] Getting public key of CryptoKeyVersion: %#v", url)
-		res, _ = sendRequest(config, "GET", cryptoKeyId.KeyRingId.Project, url, nil)
+
+		err = retryTimeDuration(func() error {
```

Comment:
you should be able to specify the error retry pred and timeout using `sendRequestWithTimeout` instead, and you won't need this retryTimeDuration